pkg/api/dashboard_snapshot.go

```diff
@@ -128,7 +128,8 @@ func (hs *HTTPServer) CreateDashboardSnapshot(c *models.ReqContext) response.Res
 			}
 		}
 
-		url = setting.ToAbsUrl("dashboard/snapshot/" + cmd.Key)
+		// Fork: Use iframe parent location for snapshot URL
+		url = cmd.AppUrl + "/dashboard/snapshot/" + cmd.Key
```

Comment:
Should I still include the code that fixes the snapshot URL despite us removing the snapshot functionality? It would be easier to just add the snapshot functionality back than go back through and remember how to implement this.

Reply:
I’d be fine with leaving it (I think that’s what you’re suggesting is preferable here, from your perspective).